spec/javascripts/stimulus/dropdown_controller_test.js

```diff
@@ -23,6 +23,10 @@ describe("Dropdown controller", () => {
       </div>`;
     });
 
+    afterEach(() => {
+      document.body.innerHTML = "";
+    });
+
```

Comment:
I'm guessing that Jest automatically cleans up the body for us after each test. So.. why was this here before do you know?

Reply:
I don't have a clear answer ... I notice that this modification actually solves the issue, and that's why I didn't look into it so much.